terra-jupyter-aou/Dockerfile

```diff
@@ -121,28 +124,12 @@ RUN mkdir -p /tmp/bcftools && \
     cd $HOME && \
     rm -rf /tmp/bcftools
 
-## Environment Configuration
 
-RUN echo "Sys.setenv(RETICULATE_PYTHON = '$(which python3)')" >> ~/.Rprofile
-
-# Install Notebook libraries as the user.
-
-ENV USER jupyter-user
+ENV USER jupyter
 USER $USER
 
 RUN R -e 'BiocManager::install(c( \
   "SAIGEgds", \
   "GENESIS"))'
 
-# Install a global gitignore file that denies all by default and then specifcally allows
-# code and documentation files.
-# https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer
-COPY --chown=$USER:users gitignore_global /home/$USER/gitignore_global
```

Comment:
@deflaux @calbach I removed these lines that writing files to `$HOME` dir....because the `gcr.io/deeplearning-platform-release/base-cu110` image we base on declares `$HOME` as a volume, so anything written to the directory in `Dockerfile` won't actually exist in the container...AoU might want to move these cmds to start up script when upgrading image in the future

Reply:
@Qi77Qi It's wonderful that the Terra detachable persistent disk will now be mounted under `$HOME`!!!

Now that these commands will be shifted to the startup script, please be sure to delete the corresponding tests I added to `terra-jupyter-aou/tests/aou_smoke_test.ipynb` via https://github.com/DataBiosphere/terra-docker/pull/234

Reply:
>@Qi77Qi It's wonderful that the Terra detachable persistent disk will now be mounted under $HOME!!!

Oh, just to be clear...we have a ticket for that, but we haven't committed to do it yet....


> Now that these commands will be shifted to the startup script, please be sure to delete the corresponding tests I added to terra-jupyter-aou/tests/aou_smoke_test.ipynb via #234


Sounds good! I have some other things that I try to finish up right now..will get to this PR soon after that